uint16 counter
uint16 time
int16 Heading_psi
uint16 Heading_psi_conf
int16 Bearing_gamma
int16 Distance
uint16 Bearing_gamma_conf
uint16 Distance_conf
uint16 Fill1
uint16 Fill2
int16 Body_pitch
int16 Body_roll
uint16 Body_pitch_conf
uint16 Body_roll_conf
int16 Ground_pitch
int16 Ground_roll
uint16 Ground_pitch_conf
uint16 Ground_roll_conf
int16 Gimbal_angle1
int16 Gimbal_angle2
int16 Gimbal_angle3
int16 Velocity
int16 Pitch_rate
int16 Yaw_rate
int16 Roll_rate
uint16 Mode
uint8 status